src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets

```diff
@@ -237,15 +222,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuild
       Projects="@(ValidProjectInputForPackGraph)"
       Targets="_AddMetadataToProjectReference"
-      Properties="%(_MSBuildProjectReferenceExistent.SetConfiguration);
-              %(_MSBuildProjectReferenceExistent.SetPlatform);
-              CustomAfterMicrosoftCommonTargets=$(MSBuildThisFileFullPath);
+      Properties="CustomAfterMicrosoftCommonTargets=$(MSBuildThisFileFullPath);
               BuildProjectReferences=false;
-              TargetFramework=$(TargetFramework);
+              _ParentTargetFramework=$(TargetFramework);
```

Comment:
Ditto for IncludeAssets, ExcludeAssets, PrivateAssets: no need to pass them back and forth. Now that I've gone through this mental refactoring, I see all we really want from the reference that we don't already have are PackageId and PackageVersion. :)